### DISCUSSION TITLE
Eslint  and type script issue  on props  in shad cn drawer component

### DISCUSSION THREAD

**Discussion by hashim7708** [shadcn-ui/ui#3756] [General]
9:3  error  'shouldScaleBackground' is missing in props validation  react/prop-types
  after running  eslint (npm run lint)  it shows this error
![Drawer tsx - frontend - Visual Studio Code 5_16_2024 6_53_56 PM](https://github.com/shadcn-ui/ui/assets/100937396/cffa8d3d-043e-4545-b3be-5d99ca09a4e9)

**Comment by Aman-Singh-Kushwaha** [comment]
I am also getting multiple lint errors :
`toast`, `textarea`, `input`, `calendar` components

![image](https://github.com/user-attachments/assets/2ba2d460-ff2e-4774-ab5d-67b1d05482a2)

I had installed using `pnpm exec shadcn@latest add toast textarea input calender`

Do we need to raise issue for lint error corrections , we is there some way to tackle it ?

Also, is it common to all ?
